app/models/email.rb

```diff
@@ -1,3 +1,11 @@
+# == Schema Information
```

Comment:
I was not sure either, but wanted to stay consistent with the other models, also the comment does not refer to a database_table, it just lists the attributes that the model has.